marklogic-data-hub/src/test/java/com/marklogic/hub/MappingE2E.java

```diff
@@ -135,6 +135,9 @@ public void setupEach() {
             }
             createMappings();
```

Comment:
This file has been changed in https://github.com/marklogic/marklogic-data-hub/pull/1202 and these changes are included in that PR.